src/components/Table/TableBody/RowActionsCell/RowActionsCell.jsx

```diff
@@ -218,7 +218,11 @@ class RowActionsCell extends React.Component {
                       key={`${tableId}-${id}-row-actions-button-${actionId}`}
                       data-testid={`${tableId}-${id}-row-actions-button-${actionId}`}
                       kind="ghost"
-                      hasIconOnly={!labelText}
+                      /** This can be changed to hasIconOnly once https://github.com/IBM/carbon-addons-iot-react/issues/1346
+                      is fixed */
+                      className={classnames({
+                        [`${iotPrefix}--row-actions-cell-btn--icononly`]: !labelText,
+                      })}
```

Comment:
I think we just need to add `tooltipPosition="bottom"` to the `Button` so it receives the tooltip positioning class(es).

Reply:
Moved it to the left because if its at the bottom or top, it still renders the space for the tooltip, adding extra height and a scrollbar to a single row